perl/Perl4-CoreLibs.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/perl/Perl4-CoreLibs" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>Perl4-CoreLibs</name>
+  <summary xml:lang="en"> libraries historically supplied with Perl 4</summary>
+  <description xml:lang="en">This is a collection of &quot;.pl&quot; files that have historically been bundled
+with the Perl core but are planned not to be so distributed with
+core version 5.15 or later.  Relying on their presence in the core
+distribution is deprecated; they should be acquired from this CPAN
+distribution instead.  From core version 5.13, until their removal,
+it is planned that the core versions of these libraries will emit a
+warning when loaded.  The CPAN version will not emit such a warning.
+</description>
+  <icon href="http://repo.roscidus.com/perl/perl.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="http://repo.roscidus.com/perl/perl.png" type="image/png"/>
+  <category>Development</category>
+  <homepage>https://metacpan.org/pod/Perl4::CoreLibs</homepage>
+  <needs-terminal/>
+  <implementation id="sha1new=b36e6483c2da3e2d1cc6913120ed9c2afa5dbaca" license="Perl License" released="2017-07-29" version="0.004">
+    <archive href="https://cpan.metacpan.org/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
+    <archive href="ftp://ftp.cpan.org/pub/CPAN/modules/by-authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
+    <archive href="http://search.cpan.org/CPAN/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
```

Comment:
Please note that 0install currently does not automatically try the next `<archive>` in the list if the first one fails. It will always pick the first `<archive>` with a MIME type it understands. But this may change in future, so it should be fine to leave the additional `<archive>`s in the feed.

Reply:
I could have sworn it did that already, hopefully it will sometime soon.